bookserver/config.py

```diff
@@ -48,6 +56,15 @@ class Settings(BaseSettings):
     dev_dburl: str = "sqlite+aiosqlite:///./runestone_dev.db"
     test_dburl: str = "sqlite+aiosqlite:///./runestone_test.db"
 
+    # Determine the database URL based on the ``config`` and the dburls above.
+    @property
```

Comment:
Does this still let you set the DB URLS in the environment?

I think it is critical to be able to continue to do that. Especially when we move to postgres and have embedded passwords.

Reply:
Yes -- the only unsettable item is `database_url`, which is set based on the other variables instead of being set directly.

Reply:
Oh right, this is a net add of an item the other database url variables are still there.